docs/modules/components/pages/processors/ollama_chat.adoc

```diff
@@ -0,0 +1,123 @@
+= ollama_chat
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+
+
+////
+     THIS FILE IS AUTOGENERATED!
+
+     To make changes, edit the corresponding source file under:
+
+     https://github.com/redpanda-data/connect/tree/main/internal/impl/<provider>.
+
+     And:
+
+     https://github.com/redpanda-data/connect/tree/main/cmd/tools/docs_gen/templates/plugin.adoc.tmpl
+////
+
+
+component_type_dropdown::[]
+
+
+Processor that uses the Ollama API to generate text.
```

Comment:
```suggestion
Sends prompts to your chosen Ollama large language model (LLM) and generates text from the responses, using the Ollama API.
```
I wondered if we could give the user a more detailed description of what the processor does. This is just a suggestion.

Reply:
> Also, both processors in the PR need adding to the nav.adoc to show up in the left-hand navigation.

Where is nav.adoc? I don't know where that is

Reply:
I'll need to do this in the other repo but it's just a reminder because needs to go out at the same time. This caught me out on Friday.